conanfile.py

```diff
@@ -64,7 +64,7 @@ def package_info(self):
         common_flags = ["-isysroot%s" % sysroot]
 
         if self.settings.get_safe("os.version"):
-            common_flags.append(tools.apple_deployment_target_flag(self.settings.os, self.settings.os.version))
+            common_flags.append(tools.apple_deployment_target_flag(self.settings.os, self.settings.os.version, self.settings.os.sdk))
```

Comment:
It seems this one is a `kwarg`: https://docs.conan.io/en/latest/reference/tools.html#tools-apple-deployment-target-flag

Reply:
@Linux13524 could you replace with `os_sdk=self.settings.os.sdk`?